### DISCUSSION TITLE
How to aply Styles to make it work in ios?

### DISCUSSION THREAD

**Discussion by Daantje007** [unoplatform/uno#9095] [Q&A] [answered]
I have tried a few differend style aprouches (app/page/ and inside control styles), for the ListViewItem but i cant get them to work in ios. What could be missing? it get just invisible, or when whole ListView style is aplied it wont even navigate to the page hihi..
But style for the PivotItems worked just fine.

**Comment by Daantje007** [comment]
I have seen this, which says i can basicly do anything with style. are there limitations for some controls? 

https://www.youtube.com/watch?v=pL2pYF7b4AY&ab_channel=UnoPlatform

**Reply by jeromelaban** [reply]
Can you provide a sample of what you're trying to do ? Does [this](https://playground.platform.uno/#listview) help?

**Reply by Daantje007** [reply]
http://playground.platform.uno/#c46e6b16

**Reply by Daantje007** [reply]
I understand hovers and such could be a problem. so i have tried to narrow it down to only selected background and foreground, **still invisible in ios.**
and a style without x:Key="Name" on a verry basic list(string).  **also invisible result**

in the playground/#xxxb16 i didnt hook up one of the uwp styles yet.

**Reply by jeromelaban** [reply]
Thanks. If you can provide a simple repro app with your test, it will help troubleshoot.

**Reply by Daantje007** [reply]
Here you go havyweight mr.L! XD

Many thanks to the whole team.

**Comment by Daantje007** [comment, marked as answer]
[LIstVIewSylingJL.zip](https://github.com/unoplatform/uno/files/8970727/LIstVIewSylingJL.zip)

**Reply by jeromelaban** [reply]
Thanks for the repro! You're hitting https://github.com/unoplatform/uno/issues/1444. The best workaround at this time is to use the fully materialized ListViewItem template, like this one:
https://github.com/unoplatform/uno/blob/1cbc1340b62aa8f09128c12666f59d57ea3dc296/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/ListViewItem_themeresources.xaml#L232
